Bug Report: Unsloth Studio Frontend Build Fails on Colab

**Bug Report: Unsloth Studio Frontend Build Fails on Colab**

**Environment**
- Notebook: `Unsloth_Studio_Colab.ipynb` (official)
- Platform: Google Colab (T4 GPU)
- Node: v20.19.0 | npm: 11.13.0 | bun: 1.3.13

**Describe the Bug**
Running the setup cell clones the repo and attempts to build the frontend, but fails with exit code 2 during `npm run build`.

**Error Message**
```
error: npm run build failed (exit code 2)

src/features/chat/utils/delete-thread-message.ts(19,35): error TS2307:
Cannot find module '@assistant-ui/core/runtime/utils/message-repository'
or its corresponding type declarations.
```

**Steps to Reproduce**
1. Open the official `Unsloth_Studio_Colab.ipynb` notebook
2. Run the first cell: clone repo + setup
3. Build fails at the frontend step

**Expected Behavior**
Setup should complete successfully and Unsloth Studio should launch.

**Actual Behavior**
Frontend build fails due to a missing internal module path in `@assistant-ui/core`.

**Possible Cause**
`@assistant-ui/core` appears to have changed its internal module structure. The path `@assistant-ui/core/runtime/utils/message-repository` no longer exists in the currently installed version, causing the TypeScript compilation to fail.

**Additional Context**
This seems to be a dependency version mismatch. Pinning `@assistant-ui/core` to an older compatible version may resolve it.

When I tried installing Unsloth on my Linux machine I got the same error, so it's not Colab specific. 
```
> tsc -b && vite build

src/features/chat/utils/delete-thread-message.ts(19,35): error TS2307: Cannot find module '@assistant-ui/core/runtime/utils/message-repository' or its corresponding type declarations.
```
Distro is Fedora 43, if that helps narrow the cause down. 

node v24.15.0 | npm 11.12.1 | bun already installed (1.3.13)

--- a/studio/frontend/package.json
+++ b/studio/frontend/package.json
@@ -16,6 +16,7 @@
     "biome:fix": "biome check . --write"
   },
   "dependencies": {
+    "@assistant-ui/core": "0.1.16",
     "@assistant-ui/react": "^0.12.19",
     "@assistant-ui/react-markdown": "^0.12.3",
     "@assistant-ui/react-streamdown": "^0.1.2",

--- a/studio/frontend/package.json
+++ b/studio/frontend/package.json
@@ -16,7 +16,7 @@
     "biome:fix": "biome check . --write"
   },
   "dependencies": {
-    "@assistant-ui/core": "0.1.16",
+    "@assistant-ui/core": "0.1.17",
     "@assistant-ui/react": "^0.12.19",
     "@assistant-ui/react-markdown": "^0.12.3",
     "@assistant-ui/react-streamdown": "^0.1.2",

--- a/studio/frontend/src/features/chat/utils/delete-thread-message.ts
+++ b/studio/frontend/src/features/chat/utils/delete-thread-message.ts
@@ -16,7 +16,7 @@ import type {
  * delete + reload smoke tests; the path or API may change without a semver signal on “public”
  * surface area.
  */
-import { MessageRepository } from "@assistant-ui/core/runtime/utils/message-repository";
+import { MessageRepository } from "@assistant-ui/core/internal";
 import { db } from "@/features/chat/db";
 import type { MessageRecord } from "@/features/chat/types";
 

